src/test/scala/nl.knaw.dans.easy.solr/CustomMatchers.scala

```diff
@@ -17,22 +17,23 @@ package nl.knaw.dans.easy.solr
 
 import java.io.File
 
-import org.apache.commons.io.FileUtils._
-import org.scalatest.matchers.{MatchResult, Matcher}
+import org.scalatest.matchers.{ MatchResult, Matcher }
 
+import scala.io.Source
 
-/** See also <a href="http://www.scalatest.org/user_guide/using_matchers#usingCustomMatchers">CustomMatchers</a> */
+/** Does not dump the full file but just the searched content if it is not found.
+  *
+  * See also <a href="http://www.scalatest.org/user_guide/using_matchers#usingCustomMatchers">CustomMatchers</a> */
 trait CustomMatchers {
   class ContentMatcher(content: String) extends Matcher[File] {
-    def apply(left: File) = {
+    def apply(left: File): MatchResult = {
       def trimLines(s: String): String = s.split("\n").map(_.trim).mkString("\n")
       MatchResult(
-        trimLines(readFileToString(left)).contains(trimLines(content)),
+        trimLines(Source.fromFile(left).mkString).contains(trimLines(content)),
```

Comment:
@jo-pol I found the best way to resolve this one is to change `File` to `Path`. Since the project isn't that big, I imagine it shouldn't take much effort to convert this...